pkg/controllers/disruption/controller.go

```diff
@@ -201,7 +207,7 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command,
 	})
 	// Cordon the old nodes before we launch the replacements to prevent new pods from scheduling to the old nodes
 	if err := state.RequireNoScheduleTaint(ctx, c.kubeClient, true, stateNodes...); err != nil {
-		return multierr.Append(fmt.Errorf("tainting nodes (command-id: %s), %w", commandID, err), state.RequireNoScheduleTaint(ctx, c.kubeClient, false, stateNodes...))
```

Comment:
Why are we removing this ? 

nvm, synced offline. Requiring the taint to not exist on all taints that aren't in the orchestration queue is a pre-condition to the reconcile loop. We only taint one set of nodes per reconcile loop, so we can always rely on the loop short circuiting on failure and untainting at the beginning of the loop.